### DISCUSSION TITLE
EFI OC 0.7.1

### DISCUSSION THREAD

**Discussion by itsabhishekolkha** [itsabhishekolkha/Mi-notebook-14-Hackintosh#8] [Announcements]
# Changes:-

OC `v0.7.1`
* Added SyncTableIds quirk to sync * modified table OEM identifiers
* Added CPU Info (MSRs) dumping to SysReport
* Updated builtin firmware versions for SMBIOS and the rest
* Fixed PowerTimeoutKernelPanic on macOS 12
* Fixed transparency click * detection on OpenCanopy boot entries
* Added PCI device info dumping to SysReport
* Fixed SetApfsTrimTimeout on macOS 12
* Documented requirement for * * * SetDefault.icns width to match Selector.icns width
* Added explicit warn and safe fallback to builtin picker on failure to match the above
* Added VSCode source level IDE debug config example to debug docs
* Added other minor debug docs updates
* Fixed incorrect timeout of built-in picker on IA32
* Added support for custom kernels on ESP partition
* Fixed DEBUG ASSERT on pressing change entry keys with single boot entry in OpenCanopy
* Added recommended Apple12 and Windows11 flavours
* Added TpmInfo tool to DEBUG TPM status
* Fixed incorrect OpenCanopy initial display when default entry beyond right of screen
* Fixed ProvideCurrentCpuInfo MSR patch on macOS 12
* Fixed AppleXcpmForceBoost patch on macOS 12
# Other Changes
1. Added `restrict Events kext`
-> For blocking unwanted processes causing compatibility issues on different hardware and unlocking the support for certain features restricted to other hardware.

2. Hibernation Fixup
    *Added **BOOT arguments**-
`-hbfx-dump-nvram`- saves NVRAM to a file nvram.plist before hibernation and after kernel panic (with panic info)
`-hbfx-disable-patch-pci` - disables patching of IOPCIFamily (this patch helps to avoid hang & black screen after resume (restoreMachineState won't be called for all devices)
**now the system will hibernate instead of regular sleep in order to save more power.**
3.**Cpu power management**
   ->Power management (Native Power Management via CFG Lock)
  * Now the heating problem is solved with native CPU power management and SMBIOS changed to MacBook pro 15,4( normal 
     temperature will be 41-degree Celcius and CPU ideal frequency will be 800MHz-1.05GHz
# NOTE- **there are many more changes that aren't mentioned here.**

<hr /><em>This discussion was created from the release <a href='https://github.com/itsabhishekolkha/Mi-notebook-14-Hackintosh/releases/tag/0.7.1'>EFI OC 0.7.1</a>.</em>

**Comment by chadify** [comment]
There's only source code in release section.
